applicationbase/functest/test_utils.go

```diff
@@ -81,143 +77,3 @@ func createMember(t *testing.T) *CommonUser {
 	member.Ref = ref
 	return member
 }
-
```

Comment:
after deletion I thing `var contracts = map[string]*contractInfo{}` no more need and can be deleted

Reply:
you are absolutely right!